ui/admin/app/components/form/target/details/index.hbs

```diff
@@ -244,7 +246,35 @@
       {{/if}}
     </Hds::Form::TextInput::Field>
   {{/if}}
-
+  {{#if this.showUpdateWorkerFilterButton}}
+    {{#if form.isEditable}}
+      <Hds::Button
+        @text={{t
+          'resources.target.workers.egress_worker_filter.filter.update_filter'
+        }}
+        @color='secondary'
+        disabled={{form.disabled}}
+        {{on 'click' this.updateDeprecatedFilter}}
+      />
+    {{/if}}
+  {{else}}
+    {{#unless this.egressWorkerFilterEnabled}}
+      <WorkerDiagram::SingleFilter />
+    {{/unless}}
+    {{#if this.egressWorkerFilterEnabled}}
+      <WorkerDiagram::SingleFilter @egressFilter='true' />
+    {{/if}}
```

Comment:
Could we use `this.egressWorkerFilterEnabled` to set `@egressFilter`? We will want to eventually wrap the `WorkerDiagrams` with the feature flags for which edition it is. It should help with readability when that comes.

Reply:
done!